File: BaekJoon/10988_팰린드롬인지 확인하기.java
import java.util.*;
public class Main {
	public static void main(String[] args) {
		Scanner sc =new Scanner(System.in);		
		String word = sc.nextLine(); int j = word.length()-1;
		int check = 1;
		for (int i = 0; i < word.length()/2; i++, j--) {
			if (word.charAt(i) != word.charAt(j)) {
				check = 0;
				break;
			}
		}
		System.out.print(check);
	}
}

File: BaekJoon/10988_팰린드롬인지 확인하기.kt
import java.util.*
fun main() {
    val sc = Scanner(System.`in`)
    val n = sc.nextLine()
    var index = n.length-1
    var check = 0
    for (i in 0..n.length/2){
        if (n[i] != n[index]) {
            check = 1
        }
        index--;
    }
    if (check == 0) {
        println(1)
    }
    else println(0)
}
